shiki/WordPress-Android
# Giphy Picker: Add Empty View

![dec-10-2018 17-59-56](https://user-images.githubusercontent.com/198826/49771239-735bce80-fca5-11e8-9375-a108464a503a.gif)

This is a part of the Giphy picker feature (wordpress-mobile/WordPress-Android#8627). It adds empty views for 2 scenarios:

- When the user first opened the picker
- When the user performed a search and there are no results

## Testing

1. Create a [Giphy API key](https://developers.giphy.com/docs/) and paste it in your `gradle.properties` as the value for `wp.giphy.api_key`
2. Go to the Media Library and click on the _Add_ button at the top right. Select the _Choose from Giphy_ option.
3. Notice the empty view when the picker is shown.
4. Perform a search with a dummy string that should have no search results.
